src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -2,17 +2,17 @@ import "@polymer/paper-button/paper-button";
 import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
 import { fireEvent } from "../../../common/dom/fire_event";
 import { HomeAssistant } from "../../../types";
+import { LovelaceConfig } from "../types";
 
 let registeredDialog = false;
 
 export class HuiCardOptions extends LitElement {
   public cardId?: string;
+  public cardConfig?: LovelaceConfig;
   protected hass?: HomeAssistant;
 
   static get properties(): PropertyDeclarations {
-    return {
-      hass: {},
-    };
+    return { hass: {} };
```

Comment:
Not sure why this was changed. I dont think this is "prettier"

Reply:
It is, small objects will be put on a single line.